frontend/device/generic/device.lua

```diff
@@ -53,9 +56,7 @@ local Device = {
     isSonyPRSTUX = no,
     isSDL = no,
     isEmulator = no,
-    canRestart = yes,
-    canReboot = no,
-    canPowerOff = no,
+    isDesktop = no,
```

Comment:
Not sure how much it matters, but by the xdg-open = desktop logic I believe this means Sailfish `isDesktop() == true`. Sailfish should probably be able to run on a regular x86 desktop, but, um, well… ;-)

Reply:
nah, Sailfish canOpenLink but is not isDesktop. In fact Sailfish isAndroid.

Desktop devices are sdl that run without a sandbox. Which means everything except Ubuntu Touch.

Reply:
> Desktop devices are sdl that run without a sandbox. Which means everything except Ubuntu Touch.

Pretty sure Sailfish is more desktop than Ubuntu Touch then. ;-)